index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>US 2024 Election Results</title>
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            text-align: center;
+        }
+        #chart-container {
+            width: 80%;
+            margin: auto;
+            padding: 50px;
+        }
+    </style>
+</head>
+<body>
+    <h1>US 2024 Election Results</h1>
+    <div id="chart-container">
+        <canvas id="electionChart"></canvas>
+    </div>
+
+    <script>
```

Comment:
Maybe a separate JavaScript file "fetch_script.js" that is called to run the fetching instead. This would clean up the HTML file. 
```suggestion
    <script src="fetch_script.js"></script>
```